bindata/network/kuryr/002-crds.yaml

```diff
@@ -0,0 +1,27 @@
+apiVersion: apiextensions.k8s.io/v1beta1
```

Comment:
Take this as a TODO to add some kind of validation and pretty printing for these CRDs.

Reply:
Added to backlog. Currently we don't have that upstream either.